Gruntfile.js

```diff
@@ -95,14 +104,147 @@ module.exports = function(grunt) {
 				base: DOC_DEST
 			},
 			src: ['**']
+		},
+
+		'export': {
+			target: {
+				src: ['src/sketch/*.sketch']
+			},
+			options: {
+				platforms: ['ANDROID', 'IOS', 'WEB'],
+			},
 		}
 
 	});
 
 
 	grunt.registerTask('optimize', ['svgmin']);
+  grunt.registerTask('export', ['export']); //'clean:icons',
 	grunt.registerTask('dist', ['optimize', 'svgstore']);
 
-	grunt.registerTask('default', ['clean', 'dist', 'preprocess']);
+	grunt.registerTask('default', ['clean:all', 'dist', 'preprocess']);
 	grunt.registerTask('ghpages', ['default', 'gh-pages']);
+
+	grunt.registerTask('export_artboards', 'export artboards', function(artboardNames, src, platform) {
```

Comment:
`grunt.task.loadTasks('./tasks');` at the top of the file will allow you to move this to a separate file in `./tasks/`